ironfish-cli/src/commands/wallet/balances.ts

```diff
@@ -0,0 +1,94 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { BufferUtils, CurrencyUtils, GetBalancesResponse } from '@ironfish/sdk'
+import { CliUx, Flags } from '@oclif/core'
+import { IronfishCommand } from '../../command'
+import { RemoteFlags } from '../../flags'
+
+// TODO(rohanjadvani): Remove this after assets are added to the wallet
+type Balance = GetBalancesResponse['balances'][number] & {
+  name: string
+}
+
+export class BalancesCommand extends IronfishCommand {
+  static description = `Display the account's balances for all assets`
+
+  static flags = {
+    ...RemoteFlags,
+    all: Flags.boolean({
+      default: false,
+      description: `Also show unconfirmed balance, head hash, and head sequence`,
+    }),
+    confirmations: Flags.integer({
+      required: false,
+      description: 'Minimum number of blocks confirmations for a note',
+    }),
+  }
+
```

Comment:
Do you think this should live in a separate command? Seems like this could also be a flag on the `wallet:balance` command

Reply:
I would say, consider the implication of that suggestion outside of this command and how to be consistent. Are you implying no pluralized command should exist? Should we merge `wallet:transactions` into `wallet:transaction`? `wallet:transactions` shows you a summary of each TX, `wallet:transaction` will dump the raw full data TX

I think it's good to have smaller very clearly purposed commands, which is the unix mantra.

Reply:
Yeah. Balance for other blockchains (like Ethereum) generally includes a specific asset, so I chose to use the plural to indicate the command is across all assets. Open to changing it, though.